hera_mc/cm_health.py

```diff
@@ -15,15 +15,19 @@
 
 def check_for_overlap(interval):
     """
+    Checks to see if two intervals overlap.  Notionally for time, but will work
+    for any ints/floats.
+
     Parameter
     ---------
-    intervals:  gps_time intervals to test for overlap
-                format [[span_1_low, span_1_high], [span_2_low, span_2_high]]
-                if the high values are None, they get set to the max value + 100 sec
+    interval :  intervals to test for overlap
+               format [[span_1_low, span_1_high], [span_2_low, span_2_high]]
+               if the high values are None, they get set to the max value + 100 sec
 
     Return
     ------
-    boolean:  True of overlap
+    boolean
+                True of overlap
```

Comment:
What does this mean? I thought it meant it would return True if there was an overlap found. I find the word "of" confusing...